app.rb

```diff
@@ -9,4 +11,21 @@ class MakersBnb < Sinatra::Base
   get '/' do
     'Hello world!'
   end
+
+  get '/add_booking' do
+    erb :add_booking
+  end
+
+  get '/booking/date_selection' do
+    @dates = Bookings.print_dates
+    @checked_availability = Bookings.check_availability(@dates)
+    erb :booking_date_selection
+  end
```

Comment:
I think we need to fully decide the convention we want to use for '/....' as we put '/makersbnb/...' and the other just has '/.../' (and this one has '/booking/...'